src/Control/Carrier/Resumable/Resume.hs

```diff
@@ -46,6 +47,12 @@ instance MonadTrans (ResumableC err) where
   lift = ResumableC . lift
   {-# INLINE lift #-}
 
+instance MonadUnliftIO m => MonadUnliftIO (ResumableC err m) where
+  askUnliftIO = ResumableC $ withUnliftIO $ \u -> return (UnliftIO (unliftIO u . runResumableC))
+  {-# INLINE askUnliftIO #-}
+  withRunInIO inner = ResumableC $ withRunInIO $ \run -> inner (run . runResumableC)
+  {-# INLINE withRunInIO #-}
+
```

Comment:
(Another heads up in case of merge conflicts: `ResumableC` is going away in #269.)

Reply:
Ported this to `fused-effects-resumable`.